modules/v2x/v2x_proxy/obu_interface/grpc_interface/grpc_client.cc

```diff
@@ -21,30 +21,31 @@
 
 #include "modules/v2x/v2x_proxy/obu_interface/grpc_interface/grpc_client.h"
 
+#include <chrono>
+
 #include "cyber/common/log.h"
+#include "cyber/cyber.h"
 
 namespace apollo {
 namespace v2x {
 
-using apollo::perception::PerceptionObstacles;
+using ::apollo::obu::v2x::CarStatus;
+using ::apollo::obu::v2x::UpdateStatus;
+using ::apollo::obu::v2x::V2X_IntersectionTrafficLightData;
+using ::apollo::obu::v2x_perception::V2X_PerceptionObstacles;
 using grpc::Channel;
 using grpc::ClientContext;
 using grpc::Status;
 
 GrpcClientImpl::GrpcClientImpl(std::shared_ptr<Channel> channel)
-    : stub_(CarToObu::NewStub(channel)) {
+    : stub_(::apollo::obu::v2x::CarToObu::NewStub(channel)) {
   AINFO << "GrpcClientImpl initial success";
```

Comment:
About namespace. We expect apollo::obu be for a separate directory `modules/obu`.
If obu is a component within `v2x` then `apollo::v2x::obu` should be used.